modules/ROOT/pages/6.0-release-notes-premium-changes.adoc

```diff
@@ -116,15 +282,24 @@ Table of Contents has been updated and is now a https://tiny.cloud/tinymce/featu
 
 Users of the Free version of {productname} 5.x who rely on the, now deprecated and removed, `toc` plugin, should not upgrade to {productname} 6.0 without a https://tiny.cloud/pricing/[subscription plan] that provides access to Premium plugins such as https://tiny.cloud/tinymce/features/#productivity[_Table of Contents_].
 
-The new Premium `tableofcontents` plugin also changes the names of several commands.
+The new Premium `tableofcontents` plugin also changes the names of several elements.
 
-* `toc` is now `tableofcontents`
-* `tocupdate` is now `tableofcontentsupdate`
-* `toc_class` is now `tableofcontents_class`
-* `toc_depth` is now `tableofcontents_depth`
-* `toc_header` is now `tableofcontents_header`
+[cols="1,1"]
```

Comment:
Perhaps add `element type` here as you did with spelling - there are a few element types in this list as listed at the top of the file.

Reply:
`Element type` column added. Two questions emerged when putting the new column together.

1. What is `tableofcontentsupdate`?

2. Are the other elements correct?

Reply:
It's a toolbar item, this is already listed at the top of the file (line 43) :)

Reply:
::facepalm::

Also: thanks for pointing me to my own work. It’s not less mortifying, but it is appropriately mortifying. :-)